internal/models/queue.go

```diff
@@ -8,19 +8,19 @@ import (
 type QueueId Id
 
 type Queue struct {
-	// Unique ID for the queue. 
+	// Unique ID for the queue.
 	Id QueueId
 
-	// Name of the queue. The name is used in generating queue urls, for ex 
-	// https://simplq.me/j/<QueueName>. The storage layer guarantees sure that 
+	// Name of the queue. The name is used in generating queue urls, for ex
+	// https://simplq.me/j/<QueueName>. The storage layer guarantees sure that
 	// there is only one queue by a given name.
 	QueueName string
 
 	// Set to true if the queue is temporarily not issuing tokens
-	bool IsPaused
+	IsPaused bool
```

Comment:
Oh. I had compiled it before checkin, and it worked because the file was never imported anywhere I think.

Reply:
Yeah